pipeline/tasks/gapic_tasks.py

```diff
@@ -130,11 +130,10 @@ def execute(self, final_repo_dir, gapic_code_dir):
 class GapicPackmanTask(packman_tasks.PackmanTaskBase):
     default_provides = 'package_dir'
 
-    def execute(self, language, api_name, final_repo_dir):
+    def execute(self, language, api_name, final_repo_dir, skip_packman=False):
         # Some APIs will be a part of gcloud project for Ruby and NodeJS.
         # Such APIs don't need packman.
```

Comment:
(I could have edited to mention that this depends on presence/absence of handwritten layers, but it seems unhelpful to document Ruby/NodeJS-specific procedure in a generic artman function.)